bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/persistence/PersistenceResource.java

```diff
@@ -58,18 +68,22 @@
  * store
  *
  * @author Chris Jackson - Initial Contribution
+ * @author Chris Jackson - Updated to add support for ModifiablePersistenceService
+ *
  */
 @Path(PersistenceResource.PATH)
 @Api(value = PersistenceResource.PATH)
 public class PersistenceResource implements RESTResource {
 
     private final Logger logger = LoggerFactory.getLogger(PersistenceResource.class);
     private final int MILLISECONDS_PER_DAY = 86400000;
+    private final SimpleDateFormat isoDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
```

Comment:
You could also refer to 'DateTimeType.DATE_PATTERN_WITH_TZ_AND_MS' if we want to make sure that we use a similar format in both cases.